src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/LogsPage/Blocks/table.html.twig

```diff
@@ -68,6 +69,23 @@
         <button type="submit" class="btn btn-primary">
           <i class="material-icons">search</i>{{ 'Search'|trans({}, 'Admin.Actions') }}
         </button>
+        {% if should_reset %}
```

Comment:
this is this block that I need to make reusable and part of Grid system :)

Reply:
'Search' button should be gone from template i think, as this comes from column filter type. :)

Reply:
this template is not used anymore (for the record)